examples/example_radionuclides.py

```diff
@@ -19,89 +19,140 @@
 
 # Norkyst
 #reader_norkyst = reader_netCDF_CF_generic.Reader(o.test_data_folder() + '/14Jan2016_NorKyst_z_3d/NorKyst-800m_ZDEPTHS_his_00_3Dsubset.nc') 
-reader_norkyst = reader_netCDF_CF_generic.Reader('http://thredds.met.no/thredds/dodsC/sea/norkyst800m/1h/aggregate_be')
-#reader_bokna = reader_ROMS_native.Reader('/home/magnes/projects/MARTINI/data/norfjords_160m_his.nc4_2015110?01-2015110?00')
+#reader_norkyst = reader_netCDF_CF_generic.Reader('http://thredds.met.no/thredds/dodsC/sea/norkyst800m/1h/aggregate_be')
 
 #o.add_reader([reader_norkyst, reader_arome])
-o.add_reader([reader_norkyst])
-#o.add_reader([reader_bokna])
-
-
+#o.add_reader([reader_norkyst])
 
-# release radionuclides at defined position and time
 #time = reader_norkyst.start_time
 td=datetime.today()
-time = datetime(td.year, td.month, td.day,0)
-#time = datetime(2015,11,2,1)   # Boknafj
+#time = datetime(td.year, td.month, td.day,0)
 
-latseed= 61.2; lonseed= 4.3    # Sognesjen
+#latseed= 61.2; lonseed= 4.3    # Sognesjen
 #latseed= 59.0;   lonseed= 10.75 # Hvaler/Koster
+
+#  Boknafjorden case
+#reader_bokna = reader_ROMS_native.Reader('/home/magnes/projects/openDrift-RAD/data/norfjords_160m_his.nc4_2015110?01-2015110?00')
+#o.add_reader([reader_bokna])
+#time = datetime(2019,9,2,1)   # Boknafj
+#time = datetime(2015,11,2,1)   # Boknafj
 #latseed= 59.486; lonseed= 5.883   # Boknafjorden, Vikedal
 #latseed= 59.09; lonseed= 5.39   # Boknafjorden, Kvitsoy
 
+# Sandnesfjorden case
+#reader_global = reader_global_landmask.Reader()
+#reader_sandnesfj = reader_ROMS_native.Reader('/home/magnes/projects/openDrift-RAD/data/norfjords_32m_his.nc4_20080???01-2008????00')
+#o.add_reader([reader_sandnesfj])
+#time = datetime(2008,4,21,1)   # Sandnesfj
+#latseed= 58.6872; lonseed= 9.0919  # Laget, Sandnesfjorden, Risr
+#latseed= 58.6825; lonseed= 9.0717  # Laget, Sandnesfjorden, Risr
 
-ntraj=10000
-#init_speciation = np.random.randint(o.nspecies,size=ntraj)
-init_speciation = np.ones(ntraj)*0
-
-diam=np.zeros_like(init_speciation,dtype=np.float32)
-#diam[init_speciation==o.name_species.index('Particle reversible')] = 0.001
-#diam[init_speciation==o.name_species.index('Particle irreversible')] = 0.001
-#print 'diam',diam
 
+# Sandnesfjorden 2 case
+reader_sandnesfj = reader_ROMS_native.Reader('/home/magnes/projects/openDrift-RAD/data/norfjords_32m_his.nc4_20180???00-2018????23')
```

Comment:
This file is only available on Magne's machine, and example will not work for other people. This is not a problem now during development, but eventually it would be good that examples work for all users. However, it is probably not so easy to make nice examples based on the small/space example ocean datasets included.